src/App.tsx

```diff
@@ -1,18 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
-
+import { Pagination } from './components/Pagination';
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Do not disable the linter, but try to solve the problem

Reply:
I dont understand what the problem